### PR TITLE
create review branch

### PR DESCRIPTION
**Pull request by Mahamadou-Nouridine**
in this PR i've done these tasks:

- create a new branch for the Pr
- added a new line in order to have difference with last commit

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +34,7 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
         <option value="other">Other</option>
       </select>
     </div>
+
```
Comment (Mahamadou-Nouridine) [inline index.html:42]: Please do not leave more than one empty line between HTML elements. Many newlines make the code not clean

**Review by Mahamadou-Nouridine — COMMENTED** [review]
# Changes required ♻️ 

Please check my comment below and make all the required changes

Submit for a re-review when you're done

thanks  :)

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Mahamadou-Nouridine 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
- [x] Good job using inline comments.
 
## Required Changes ♻️
- [ ] Dear @Mahamadou-Nouridine, I like your review, but we need to have no less than two review comments on the project `Please wrote at least 2 comments in the review.`. I can see you have only one. Please consider making a re-review with two comments.
_Check the comments under the review._

## Optional suggestions
- [Optional] Please consider adding an emoji/gif to your review summary. This will make your review more friendly.
- [Optional] Please also try to summarize your review in more detail. Remember this is where you give a general summary of your review. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,5 +1,149 @@
...
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 [Title] <a name="about-project"></a>
```
Comment (Mahamadou-Nouridine) [inline README.md:27]: Please use your project title instead of the default title, because the repository title can not to be the

File: README.md [hunk 3]
```diff
@@ -1,5 +1,149 @@
...
+
+**My Portfolio** is a a website that shows my achievements, my skills, my contact details and all the informations needed by a recruiter or a collaborator
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
```
Comment (Mahamadou-Nouridine) [inline README.md:33]: Please Make sure to include the tech stack used to build the project

File: index.html [hunk 4]
```diff
@@ -14,7 +14,11 @@
 
   <h1 id="title">freeCodeCamp Survey Form</h1>
   <p id="description">Thank you for taking the time to help us improve the platform</p>
+<!-- <div>
+  <h1>
 
+  </h1>
+</div> -->
```
Comment (Mahamadou-Nouridine) [inline index.html:21]: Please don't leave unused code in the code base, this makes the code not clean and long

**Review by Mahamadou-Nouridine — COMMENTED** [review]
STATUS: REQUIRED CHANGES ♻️♻️

Hi @Mahamadou-Nouridine 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f632e74656e6f722e636f6d2f764f4c575f55514d79534941414141432f616c6d6f73742d74686572652e676966

# To highlight 🍾

- Good commit history ✔️
- Your design looks good 💯 ✔️
- Good PR description ✔️
- Follow GitHub flow ✔️

**Comment by GedeonTS** [comment]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @Mahamadou-Nouridine ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- Good job using GitHub review feature :+1: 
- Polite and clear suggestions :+1: 
- Good use of inline comments :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**